basestar-event/src/test/java/io/basestar/event/TestEventSerialization.java

```diff
@@ -67,6 +98,53 @@ void testGzipBson() {
         assertNotNull(event.getAfter());
         final byte[] outputBytes = EventSerialization.gzipBson().serialize(event);
         final String output = BaseEncoding.base64().encode(outputBytes);
-        assertEquals(input, output);
+
```

Comment:
I clearly went the long route here. 
Upon re-reading, it seems it has less to do with Apple M1 than with JDK17 using a more recent version of the gzip library, which drops the out of date "OS" header field. 

@arcneon, do you want me to simplify this, keeping only the "nuke 10th byte" logic?

Reply:
we shouldn't really be testing this TBH, it's only important that all instances in the same cluster can deserialize their own messages, not important that they work between JVMs.

Reply:
so according to this the test should actually be "serialize a message / deserialize it on the way out"

(I think I disagree though; we could have a need for being able to upgrade the JDK of components of a cluster without having first to drain all SQS queues.)